### PR TITLE
Significant updates to onprem UI

### PR DESCRIPTION
**Pull request by jimmyjayp**
Enhancemnts include support for adding new cards in accordions, event monitor service on the server with socket io broadcast of status heartbeat to browser, all config screens wired to the state, lots of core work and structuring.

**This PR Addresses:**  
[schema updates to support initial deployment of onprem admin](https://trello.com/c/X73orVIA/4860-schema-updates-to-support-initial-deployment-of-onprem-admin)

### CODE REVIEW

File: onprem_admin/lib/systemStatus.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+import Fs, { watch } from 'fs';
+import Hjson from 'hjson';
+import { worstStatus, SystemStatuses } from '../src/store/actions/status';
+
+const WatchInterval = 10000;
+const watchers = {
+	fauxStatusFile: {
+		type: 'file',
+		name: `${process.env.OPADM_TMP}/fauxStatusFile.json`,
+		// lastCheck: Date.now(),
```
Comment (bcanzanella) [inline onprem_admin/lib/systemStatus.js:12]: will this fake status file stay?

File: onprem_admin/src/components/lib/Accordion.js [hunk 2]
```diff
@@ -2,13 +2,77 @@
...
+					src={cardStatus.icon}
+					onClick={
+						cardStatus?.onClickAction &&
+						((e) => {
+							console.debug('clicked the accordion header content image');
+							dispatch({ type: cardStatus.onClickAction, payload: cardStatus.onClickActionPayload });
+							// FIXME: this don't work!! - confirmed that this IS in fact, stopping
+							// the event from bubbling up, but it IS NOT stopping the card from
+							// opening or closing
+							// e.stopPropagation();
```
Comment (bcanzanella) [inline onprem_admin/src/components/lib/Accordion.js:68]: FIXME

**Review by bcanzanella — APPROVED** [review]
LGTM unless you had some specific areas you wanted to focus on?

File: onprem_admin/src/components/lib/Accordion.js (hunk 2 above)
Comment (bcanzanella) [reply onprem_admin/src/components/lib/Accordion.js:68]: (this was a test)

File: onprem_admin/lib/systemStatus.js (hunk 1 above)
Comment (jimmyjayp) [reply onprem_admin/lib/systemStatus.js:12]: no.  I haven't done any of the coding that pertains to the operating environment. Much of that needs to be determined &amp; implemented. I've just been focused with learning react/redux &amp; browser coding thus far.

File: onprem_admin/lib/systemStatus.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
+
+import { timingSafeEqual } from 'crypto';
+import Fs, { watch } from 'fs';
+import Hjson from 'hjson';
```
Comment (jimmyjayp) [inline onprem_admin/lib/systemStatus.js:4]: Testing this comment for line 4 (hjson)

**Pull request merged**